src/zendesk.ts

```diff
@@ -227,14 +229,24 @@ namespace Zendesk {
     updated_at?: string
   }
 
+  export interface GroupMembership {
+    created_at?: string
+    updated_at?: string
+    default?: boolean
+    group_id?: number
+    id?: number
+    url?: string
+    user_id?: number
+  }
+
   export interface User {
     id?: number
     email?: string
     external_id?: string
     name?: string
     phone?: string
     tags?: string[]
-    user_fields?: { [key: string]: any }
+    user_fields?: { [key: string]: UserField }
```

Comment:
`UserField`? These are scalars only, no?

Reply:
You're right updated